components/scream/src/share/tests/grid_tests.cpp

```diff
@@ -29,6 +29,8 @@ TEST_CASE("point_grid", "") {
   REQUIRE(grid->get_num_local_dofs()  == num_local_cols);
   REQUIRE(grid->get_num_global_dofs() == num_global_cols);
   REQUIRE(grid->is_unique());
+  REQUIRE(grid->get_global_min_dof()==0);
```

Comment:
is it possible to add a multi-rank test for say 2 ranks (when the machine makes it possible) that ensures that rank 1 has dofs [0,...,N/2-1] and rank 2 has dofs [N/2,...,N-1]?

Reply:
We can definitely add multi-rank tests for grids, it might help test a few more things. That said, we don't _need_ the point grid to have dofs partitioned that way, meaning that there should not be any place in the code that assumes that.

To be fair, we also don't assume that the min gid is 0, so maybe we should check that `max_gid-min_gid==num_global_gids`, which is the property we actually _need_ in IO.